styles.css

```diff
@@ -0,0 +1,158 @@
+html {
+	font-family: "Trebuchet MS", sans-sarif;
+	box-sizing: border-box;
+	font-size: 10px;
+}
+
+body {
+	background-color:#0a415c;
+	margin: 0;
+	padding: 0;
+	color:#cfd2d4;
+}
+
+h2 {
+	font-size: 3rem;
+}
+
+h3 {
+	font-size: 2rem;
+}
+
+p {
+	font-size: 1.5rem;
+}
+
+nav {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: nowrap;
+	justify-content: space-between;	
+	align-items: center;
+}
+
+.nav-bar {
+	width: 20%;
+	display: flex;
+	flex-direction: row;
+	flex-wrap: nowrap;
+	gap: 20px;
+	font-size: 2rem;
+	padding: 20px;
+}
+
+.nav-bar a:link, .nav-bar a:visited {
+	color: #cfd2d4;
+}
+.nav-bar a:hover {
+	color:#048230;
+}
+
+
+.cv-download-button {
+	width: 150px;
```

Comment:
the width of the button should be determined by the width of the text + any padding, so you shouldn't have to specify width. i recommend instead giving it padding (and while you're at it, you can do the same with the height)